### PR TITLE
RIA-3876:Out of country screening questions

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3876:Out of country screening questions

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3876

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by makowalski — APPROVED** [review]
LGTM

File: e2e/features/stepDefinitions/service/start-appeal.steps.ts [hunk 1]
```diff
@@ -4,12 +4,24 @@ import { Given, Then } from 'cucumber';
...
+const isOutOfCountryEnabled = require('../../../ia.conf').isOutOfCountryEnabled === 'true';
 const ccdFormPage = new CcdFormPage();
 const startAppealFlow = new StartAppealFlow();
 
 Given('I complete the `Tell us about your client` page', async function () {
     expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);
-    await startAppealFlow.completeScreeningQuestions(true);
+    if (isOutOfCountryEnabled) {
+        await startAppealFlow.completeScreeningQuestionsOutOfCountry(true);
+        expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);
```
Comment (Lee365) [inline e2e/features/stepDefinitions/service/start-appeal.steps.ts:15]: expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);

The above line is repeated.
The logic here need not change. This logic can be moved to start-appeal.flow.ts completeScreening Questions

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: e2e/features/stepDefinitions/service/start-appeal.steps.ts (hunk 1 above)
Comment (divisathyan) [reply e2e/features/stepDefinitions/service/start-appeal.steps.ts:15]: The line is repeated in 2 pages

Comment (divisathyan) [reply e2e/features/stepDefinitions/service/start-appeal.steps.ts]: Code moved to startAppeal.flow.ts

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by jkumar-moj — APPROVED** [review]

**Pull request merged**